morph/react-dom/get-style-for-property.js

```diff
@@ -61,7 +61,11 @@ export default function getStyleForProperty(node, parent, state, code) {
 
     case 'fontFamily':
       return {
-        fontFamily: code ? asVar(node) : maybeAddFallbackFont(node.value),
+        fontFamily: code
+          ? [asVar(node), getSystemDefaultFont()].join(',')
```

Comment:
If it is a var, does it still work to add something that isn't a var next to it? I imagine this is what it would render, is that right?
```
font-family: var(--font-family), sans-serif;
```